packages/block-library/src/blocks/supernova/editor-styles.scss

```diff
@@ -1,13 +1,40 @@
 .supernova {
   
   > .wp-block {
-    margin-top: 0;
-    margin-bottom: 0;
+  
```

Comment:
Elements with a `__inner-container` classname should be the ones which ensure sides padding.

Reply:
Also, full width blocks (hero) don't go full width inside the editor at the moment.